README.md

```diff
@@ -1,5 +1,5 @@
-Please see the [project home page](https://github.com/asciidoctor/asciidoctor-reveal.js)
-for the README.
+# reveal.js converter for Asciidoctor.js
 
-The README is written in [AsciiDoc](http://asciidoctor.org/docs/what-is-asciidoc/#what-is-asciidoc) but npmjs.com
```

Comment:
The repository https://github.com/npm/www is archived and I don't think npmjs.com is going to fix this issue anytime soon.

I think it's better to just point to the repository for the complete documentation.